pkgs/development/compilers/picat/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, gcc49, gnumake }: 
```

Comment:
I think it's cleaner to specify `stdenv = overrideCC stdenv gcc49` in `all-packages.nix` instead.